src/models/gaze16/external-overview.md

```diff
@@ -0,0 +1,122 @@
+
+#### 3060:
+
+![15" Right Ports (3060)](./img/ports-15-right-3060.webp)
+
+### Front side:
+
+![15" Front Ports](./img/ports-15-front.webp)
+
+### Back side:
+
+#### 3050 & 3050 Ti:
+
+![15" Back Ports (3050/3050 Ti)](./img/ports-15-back-3050x.webp)
+
+#### 3060:
+
+![15" Back Ports (3060)](./img/ports-15-back-3060.webp)
+
+## 17" models:
+
+### Left side:
+
+![17" Left Ports](./img/ports-17-left.webp)
+
+### Right side:
+
+#### 3050 & 3050 Ti:
+
+![17" Right Ports (3050/3050 Ti)](./img/ports-17-right-3050x.webp)
+
+#### 3060:
+
+![17" Right Ports (3060)](./img/ports-17-right-3060.webp)
+
+### Front side:
+
+![17" Front Ports](./img/ports-17-front.webp)
+
+### Back side:
+
+#### 3050 & 3050 Ti:
+
+![17" Back Ports (3050/3050 Ti)](./img/ports-17-back-3050x.webp)
+
+#### 3060:
+
+![17" Back Ports (3060)](./img/ports-17-back-3060.webp)
+
+## Lid/chassis design differences:
+
+![15"/17" design comparison](./img/gaze16-sizes.webp)
+
+## Materials:
+
+### 15" model:
+
+|Part                              |Material |
+|----------------------------------|---------|
+|LCD back cover (lid)              |Aluminum |
+|LCD front cover (bezel)           |Plastic  |
+|Top case (palm rests/port covers) |Plastic  |
+|Bottom panel                      |Plastic  |
+
+### 17" model:
+
+|Part                              |Material |
+|----------------------------------|---------|
+|LCD back cover (lid)              |Plastic  |
+|LCD front cover (bezel)           |Plastic  |
+|Top case (palm rests/port covers) |Plastic  |
+|Bottom panel                      |Plastic  |
+
+## LED indicators:
+
+The Gazelle has the following LED indicators:
+
+|Icon                                    |Color          |Description                      |
+|----------------------------------------|---------------|---------------------------------|
+|![Power LED](./img/led-power.png)       |Orange         |Powered off, DC power plugged in |
+|                                        |Green          |Powered on                       |
+|                                        |Blinking green |Sleeping (suspended)             |
+|![Battery LED](./img/led-battery.png)   |Orange         |Battery charging                 |
+|                                        |Green          |Battery fully charged            |
+|                                        |Blinking orange|Battery critically low           |
+|![Storage LED](./img/led-storage.png)   |Green          |Storage drive activity           |
+
+## Keyboard shortcuts:
+
+The Gazelle has the following actions available using the Fn and Function keys:
+
+|Key                        |Shortcut|Action                                        |
+|---------------------------|--------|----------------------------------------------|
+|![Fn-F1](./img/fn-f1.png)  |Fn+F1   |Toggle trackpad                               |
+|![Fn-F2](./img/fn-f2.png)  |Fn+F2   |Toggle built-in LCD<br/>(non-OLED models only)|
```

Comment:
Since there isn't an OLED model do we need this or just future proofing the tech-docs?

Reply:
This was left over from the oryp7 section I copied as a template, thank you for catching it. Removed with 78fd1e2.